src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -218,7 +218,33 @@ namespace HVACMultiSpeedHeatPump {
     Array1D<MSHeatPumpData> MSHeatPump;
     Array1D<MSHeatPumpReportData> MSHeatPumpReport;
 
+    bool GetInputFlag(true); // Get input flag
+    bool FlowFracFlagReady(true);   // one time flag for calculating flow fraction through controlled zone
+    int ErrCountCyc(0); // Counter used to minimize the occurrence of output warnings
+    int ErrCountVar(0); // Counter used to minimize the occurrence of output warnings
+
     // Functions
+    void clear_state() {
+        NumMSHeatPumps = 0;
+        AirLoopPass = 0;
+        TempSteamIn = 100.0;
+        CurrentModuleObject = "";
+        CompOnMassFlow = 0.0;
+        CompOffMassFlow = 0.0;
+        CompOnFlowRatio = 0.0;
+        CompOffFlowRatio = 0.0;
+        FanSpeedRatio = 0.0;
+        SupHeaterLoad = 0.0;
+        SaveLoadResidual = 0.0;
+        SaveCompressorPLR = 0.0;
+        CheckEquipName.clear();
+        MSHeatPump.clear();
+        MSHeatPumpReport.clear();
+        GetInputFlag = true; // Get input flag
+        FlowFracFlagReady = true;
+        ErrCountCyc = 0;
+        ErrCountVar = 0;
+    }
```

Comment:
Pulled a bunch of variables out from statics and added a clear_state.  Seriously how are there still files without clear state.  Crazy.